src/Checkbox/Icon.js

```diff
@@ -13,6 +13,11 @@ const icons = resolve`
         fill: ${theme.default};
     }
 
+    .focus :global(path) {
+        stroke: #646464;
+        stroke-width: 2px;
```

Comment:
Asking @cooper-joe about this.

Reply:
Ah, good catch! I completely forgot that I wanted to ask for a focus style!
@cooper-joe please help out